Library/Formula/gcc.rb

```diff
@@ -153,27 +182,14 @@ def install
       end
     end
 
-    # Handle conflicts between GCC formulae and avoid interfering
-    # with system compilers.
+    # Handle conflicts between GCC formulae.
+
     # Since GCC 4.8 libffi stuff are no longer shipped.
     # Rename man7.
     Dir.glob(man7/"*.7") { |file| add_suffix file, version_suffix }
     # Even when suffixes are appended, the info pages conflict when
     # install-info is run. TODO fix this.
     info.rmtree
-
-    # Rename java properties
-    if build.with?("java") || build.with?("all-languages")
-      config_files = [
-        "#{lib}/gcc/#{version_suffix}/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/security/classpath.security",
-        "#{lib}/gcc/#{version_suffix}/i386/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/i386/security/classpath.security",
-      ]
-      config_files.each do |file|
-        add_suffix file, version_suffix if File.exist? file
-      end
-    end
```

Comment:
I was blindly following gcc5's lead in homebrew-versions, but I guess it was probably removed there because nothing depends on gcc5, so I've restored it.